src/style.scss

```diff
@@ -189,50 +142,50 @@ slot {
     cursor: not-allowed;
     transform: unset;
 
-    background-color: rgba($auro-color-ui-default-on-light, $disabledOPacity);
-    border: 1px solid rgba($auro-color-ui-default-on-light, 0);
+    background-color: rgba($ds-color-ui-default-default, $disabledOPacity);
+    border: 1px solid rgba($ds-color-ui-default-default, 0);
 
     @media (hover: hover) {
       &:hover {
-        background-color: rgba($auro-color-ui-default-on-light, $disabledOPacity);
-        border: 1px solid rgba($auro-color-ui-default-on-light, 0);
+        background-color: rgba($ds-color-ui-default-default, $disabledOPacity);
+        border: 1px solid rgba($ds-color-ui-default-default, 0);
       }
     }
   }
 
   // auro-button--secondary
   &[variant='secondary'],
   &--secondary {
-    background-color: var(--auro-color-background-lightest);
-    border: 1px solid var(--auro-color-ui-default-on-light);
-    color: var(--auro-color-text-link-on-light);
+    background-color: var(--ds-color-background-lightest, $ds-color-background-lightest);
+    border: 1px solid var(--ds-color-ui-default-default, $ds-color-ui-default-default);
+    color: var(--ds-color-text-link-default, $ds-color-text-link-default);
 
     @media (hover: hover) {
       &:active,
       &:hover {
-        background-color: var(--auro-color-ui-bkg-hover-on-light);
-        border: 1px solid var(--auro-color-ui-default-on-light);
+        background-image: linear-gradient(var(--ds-color-ui-bkg-hover-default, $ds-color-ui-bkg-hover-default), var(--ds-color-ui-bkg-hover-default, $ds-color-ui-bkg-hover-default));
+        border: 1px solid var(--ds-color-ui-default-default, $ds-color-ui-default-default);
       }
     }
 
     &:active {
-      background-color: var(--auro-color-ui-bkg-hover-on-light);
-      border: 1px solid var(--auro-color-ui-default-on-light);
+      background-color: var(--ds-color-ui-bkg-hover-default, $ds-color-ui-bkg-hover-default);
+      border: 1px solid var(--ds-color-ui-default-default, $ds-color-ui-default-default);
     }
 
     &:disabled {
       cursor: not-allowed;
       transform: unset;
 
-      color: rgba($auro-color-text-link-on-light, $disabledOPacity);
-      background-color: rgba($auro-color-background-lightest, $disabledOPacity);
-      border: 1px solid rgba($auro-color-ui-default-on-light, $disabledOPacity);
+      color: rgba($ds-color-text-link-default, $disabledOPacity);
```

Comment:
These colors that don't use tokens are not able top be themed. Wasn't part of this work intended to get new tokens so at least all colors are theme-able?

Reply:
From my understanding, these variables are being consumed from our sass variables, so they should still be theme-able if an implementer creates their own sass files with their own values.